### DISCUSSION TITLE
Automations have stopped for no reason; are there logs?

### DISCUSSION THREAD

**Discussion by charlesdsmith** [Budibase/budibase#8773] [Help] [answered]
I have an automation that has stopped for seemingly no reason, ever since the clock went back an hour a couple weeks ago. It works when I run it manually, is there a way to view the logs for this?

**Comment by melohagan** [comment, marked as answer]
Hey @charlesdsmith 

Have you had a look at the automation history? https://docs.budibase.com/docs/logs

Otherwise you can look at the worker service to see if there's more detailed logs in there.

**Reply by charlesdsmith** [reply]
> Hey @charlesdsmith
> 
> Have you had a look at the automation history? https://docs.budibase.com/docs/logs
> 
> Otherwise you can look at the worker service to see if there's more detailed logs in there.

Hey sorry for the late reply! The automation history is blank, I looked into the service logs and saw this: it's not timestamped however so that doesnt help.

![image](https://user-images.githubusercontent.com/38652599/203697738-ecfae8ac-e541-4ca5-a600-d28ca54b6794.png)


It says unauthorized, but the role permissions are set to admin. It works when I run it manually

**Reply by melohagan** [reply]
I think that error may simply be a red herring. Do the automations resume if you re-publish your app? 
Could you provide more details/screenshots about how your automation steps are setup.

**Reply by charlesdsmith** [reply]
Republishing it did the trick! Thank you, not sure why that fixed it though

**Reply by melohagan** [reply]
Brilliant! Publishing your app pushes any changes to your automations and starts them. So if anything does go wrong, a publish can restart your automations.
